compiler/src/dotty/tools/dotc/transform/Mixin.scala

```diff
@@ -121,60 +128,70 @@ class Mixin extends MiniPhase with SymTransformer { thisPhase =>
   override def changesMembers: Boolean = true  // the phase adds implementions of mixin accessors
 
   override def transformSym(sym: SymDenotation)(using Context): SymDenotation =
```

Comment:
The documentation comment above Mixin describes the stuff that transformSym does and probably neeeds to be updated since it's doing more things now.